templates/transactions/wallet_detail.html

```diff
@@ -0,0 +1,7 @@
+<h1>{{ wallet.name }}</h1>
+<h3>User: {{ wallet.user }}</h3>
+<h3>Balance: {{ wallet.balance }}zł</h3>
+<h3>Transactions:</h3>
```

Comment:
dodaj warunek ze jak nie ma transakcji to wyswietla wiadomość ze brak transakcji